basic_rules.md

```diff
@@ -233,7 +233,7 @@ Mistrz Gry decyduje również o ilości złota otrzymanego przez postaci graczy,
 
 ### 3.2. Nagrody rzeczowe: magiczne i niemagiczne przedmioty<a name="magitki"></a>
 
-Nagrodą w przygodzie często jest jeden lub więcej magicznych przedmiotów. O ich podziale pomiędzy postacie graczy decyduje Mistrz Gry, według zasad szczegółowo opisanych w części III pliku Zasady Pełne. Generalną zasadą jest, że pierwszeństwo w uzyskaniu przedmiotu mają postacie, które mogą danego przedmiotu używać. Dodatkową regułą w wypadku permanentnych (czyli nie jednorazowego użytku) magicznych przedmiotów, o rzadkości co najmniej uncommon, jest, że pierwszeństwo w otrzymaniu takiego przedmiotu mają postacie, które posiadają takich przedmiotów mniej, niezależnie czy te przedmioty zostały zabrane na przygodę, czy nie (dlatego należy zawsze pamiętać o liczbie permanentnych przedmiotów magicznych o rzadkości co najmniej uncommon przypisanych do postaci – Mistrz Gry może potrzebować tej informacji). Do liczby magicznych przedmiotów nie wlicza się jedna magiczna, generyczna broń +1.
+Nagrodą w przygodzie często jest jeden lub więcej magicznych przedmiotów. O ich podziale pomiędzy postacie graczy decyduje Mistrz Gry, według zasad szczegółowo opisanych w części III pliku Zasady Pełne. Generalną zasadą jest, że pierwszeństwo w uzyskaniu przedmiotu mają postacie, które mogą danego przedmiotu używać. Dodatkową regułą w wypadku permanentnych (czyli nie jednorazowego użytku) magicznych przedmiotów, o rzadkości co najmniej uncommon, jest, że pierwszeństwo w otrzymaniu takiego przedmiotu mają postacie, które posiadają takich przedmiotów mniej, niezależnie czy te przedmioty zostały zabrane na przygodę, czy nie (dlatego należy zawsze pamiętać o liczbie permanentnych przedmiotów magicznych o rzadkości co najmniej uncommon przypisanych do postaci – Mistrz Gry może potrzebować tej informacji). Do liczby magicznych przedmiotów nie wlicza się jedna magiczna, generyczna broń +1. Wyjątek: przedmioty Wraps of Unarmed Prowess i Insignia of Claws o rzadkości Uncommon liczą się jako broń +1 na potrzeby tej zasady.
```

Comment:
Nie nazwałbym tego wyjątkiem, bo to rozszerza pulę przedmiotów a nie ją zawęża. Może: "...nie wlicza się pierwszy z następujących przedmiotów posiadanych przez postać: magiczna, generyczna Broń +1, Wraps of Unarmed Prowess lub Insignia of Claws (Uncommon)"?

Reply:
Poprawiłem i wrzuciłem w commit